### ISSUE TITLE
Clear backpack

### ISSUE THREAD

**Issue by ajskateboarder** [ScratchAddons/ScratchAddons#7642]
### Why this would be helpful

I have too much old junk in my backpack and/or I backpacked something a long time ago that I want to access again.

### How this addon works

This addon adds a right click item in the backpack to delete all backpacked code and assets. It should display a confirmation dialog as well.

### Possible alternatives

Alternatively, items in the backpack could be made multi-select-able with shift-click or ctrl-click so you can also delete multiple items at once.

**Comment by WorldLanguages** [comment]
Duplicate of...? I swear this has been suggested before

If we can find a good way to implement this we could, but we usually don't have features that automate network requests in behalf of the user.

**Comment by WorldLanguages** [comment]
Even griffpatch requested it!

![image](https://github.com/user-attachments/assets/e726a0f3-a47a-42d8-a9e7-6c2b7992f92b)

**Comment by WorldLanguages** [comment]
Maybe we could combine this, with similar suggestions (for example comment https://github.com/ScratchAddons/ScratchAddons/issues/6161#issuecomment-1989242536) where people have requested the ability export the backpack as a .zip file. It wouldn't be importable unless we also implement that, and I'm not sure if we could combine scripts in the backpack into a .sprite3 file easily or not. But we could allow the user to delete the backpack only after they have downloaded the backup.

**Comment by mybearworld** [comment]
> Duplicate of...? I swear this has been suggested before


This is kind of included in #7369

**Comment by LibraryOfStuff** [comment]
I'd really love this feature. My backpack is a gigantic pool of assets and code...

**Comment by mxmou** [comment]
From feedback:

> Please PLEASE add an option to clear my backpack. Like I have so much random stuff now I really need to clear. I think it'll be useful. Maybe add deleting backpack stuff after some time option too, unless you saved it? Mine's is just extremely cluttered.
